tgui/packages/tgui/interfaces/GeneTek/BioEffect.js

```diff
@@ -57,6 +57,7 @@ export const BioEffect = (props, context) => {
   } = data;
   const {
     gene,
+    showSequence=true,
```

Comment:
Guessing it complains without the `=true` bit?

Reply:
Desired behavior is to only allow removal of the DNASequence on the Mutations pane.  This ensures default behavior is to show the DNASequence unless explicitly disabled.  Previous behavior always provided DNASequence for bioEffects.

Reply:
Is there a better pattern to follow?

Reply:
Gotcha, because this is used in another place. I would unironically add a `showSequence` prop to the other usage and remove the `=true` here.